test/image-diff_test.js

```diff
@@ -67,7 +106,8 @@ describe('image-diff', function () {
     runImageDiff({
       actualImage: __dirname + '/test-files/checkerboard-excess.png',
       expectedImage: __dirname + '/test-files/checkerboard.png',
-      diffImage: __dirname + '/actual-files/different-size.png'
+      diffImage: __dirname + '/actual-files/different-size.png',
+      threshold: 0.70
```

Comment:
We should not need to update this test for threshold or similar. Please change it back.